### ISSUE TITLE
package name derived from proto file name

### ISSUE THREAD

**Issue by patriknw** [lightbend/akkaserverless-java-sdk#450]
One strange default in ScalaPB is `flat_package=false` (https://scalapb.github.io/docs/customizations/)

> Setting flat_package to true (default is false) makes ScalaPB not append the protofile base name to the package name. You can also apply this option globally to all files by adding it to your ScalaPB SBT Settings.

That means that the Scala package name becomes:
```
package customer.view.customer_view
```

@raboof mentioned that in Akka gRPC we adjust the default of `flat_package` to `true`. I think it makes sense do do the same here.

**Comment by patriknw** [comment]
I tried to add the following to the sample:

```
import "scalapb/scalapb.proto";

option (scalapb.options) = {
  flat_package: true
};
```

Surprisingly it still added the `customer_view` to the package.

**Comment by raboof** [comment]
I had a quick try disabling flat_package earlier, and for me it did have some effect, but exposed a problem with Akka gRPC and Akkaserverless generating the same classes. We probably can make sure to do this only once (taking care to pass the same scalapb options everywhere)

**Comment by patriknw** [comment]
Tried again. Ok now it has an effect (must have been IntelliJ not saving files yesterday).

I see the same problem. Generated twice:
```
target/scala-2.13/akka-grpc/main/customer/api/Customer.scala
target/scala-2.13/src_managed/main/scalapb/customer/api/Customer.scala
```

**Comment by raboof** [comment]
Fixed with #491 (though possibly it could be cleaned up a bit further)
